src/countItems.js

```diff
@@ -11,15 +11,23 @@
  * @returns {object} An object containing the count of each
  *  string in the input array
  */
-function countItems(array) {
-  // This is your job. :)
-}
+function countItems(array,value) {
```

Comment:
This isn't what the question is asking; if you're unsure what the question is asking, ask! You shouldn't have to rewrite the function definition for any of these.

Look at the example inputs and outputs.